content/english/sql_workshop/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SQL Workshop"
+description: "SQL Workshop Intro"
+date: 2019-07-25T13:24:17-07:00
+prereq: "None"
+difficulty: "Beginner"
+image: ""
+draft: false
+# icon: "fab fa-python"

```

Comment:
Please pick a new icon for this workshop based on: https://forkaweso.me/Fork-Awesome/icons/

Reply:
This one looks about right: https://forkaweso.me/Fork-Awesome/icon/database/